ocs_ci/helpers/helpers.py

```
@@ -1312,6 +1312,169 @@ def get_cephfs_data_pool_name():
+                    {
+                        "name": pool_name,
+                        "compressionMode": compression,
+                        "replicated": {"size": replica},
+                    }
+                ],
+            }
+        ]
+
+    try:
+        storage_cluster_obj.patch(params=data_pool_patch, format_type="json")
+        logger.info(f"Additional data pool '{pool_name}' added to CephFilesystem")
```

Ruff (0.15.20)

[warning] Logging statement uses f-string

(G004)